docs/usage.md

```diff
@@ -1,65 +1,109 @@
-## Usage
```

Comment:
This is live help document which will be part of the code path , we need to keep as is

Reply:
@budhac sure, just want to make sure this doc is consumed in the code path.. need to do changes if required.